code/modules/integrated_electronics/core/saved_circuits.dm

```diff
@@ -232,6 +232,9 @@
 
 	var/error
 
+	//Check for bad inputs
+	var/regex/bad_regex = new("\[^\\w{}:\",.\\-*&+ =\'\\(\\)\\\[\\\]#\]", "gi")
+	bad_regex.Replace(bad_regex, "")
```

Comment:
This doesn't do anything

Reply:
The thing should probably be put somewhere here:
https://github.com/BeeStation/BeeStation-Hornet/blob/69d83b6d9aaa9e55061f84b9efb9ba70748f316b/code/modules/integrated_electronics/core/printer.dm#L265-L272

Reply:
The regex runs on the regex itself.